tests/e2e/wdio.conf.js

```diff
@@ -149,6 +147,17 @@ export const config = {
       await browser.installAddOn(extension.toString('base64'), true);
     }
 
+    // Disable cache for Chrome to avoid caching issues for browser.url()
+    if (capabilities.browserName === 'chrome') {
+      browser.overwriteCommand('url', async function (orig, url, options) {
+        await browser.sendCommand('Network.setCacheDisabled', {
```

Comment:
that this have to be called for every navigation? I was hoping it can be done once for the browsing session

Reply:
Sadly, it works only for the "next" navigation event. I tested it when I found the solution, as I thought the same as you.

Reply:
I'm pretty sure it works for entire session. Maybe we do something wrong elsewhere?